src/screens/Bridge/Exchange-CrossChain/Exchange.tsx

```diff
@@ -101,6 +101,9 @@ function Exchange({ fetchExchangeTitle }: Props) {
   const fromInputRef: any = React.useRef();
   const screenName = getActiveScreenName(navigation);
   const colors = useThemeColors();
+  const gasFeeList = useExchangeGasFee();
+
+  const isEstimaing = gasFeeList?.find((feeInfo) => feeInfo.isEstimating);
```

Comment:
As above, can we call `.find` on a potentially `null`? At least it should be `[]`

Reply:
it's []